2024-05-31 16:01:47,968 DEBUG    root            Loaded Command Group: ['gcloud', 'sql']
2024-05-31 16:01:47,970 DEBUG    root            Loaded Command Group: ['gcloud', 'sql', 'import']
2024-05-31 16:01:48,033 DEBUG    root            Loaded Command Group: ['gcloud', 'sql', 'import', 'csv']
2024-05-31 16:01:48,036 DEBUG    root            Running [gcloud.sql.import.csv] with arguments: [--database: "[DATABASE_NAME]", --table: "[TABLE_NAME]", INSTANCE: "[INSTANCE_NAME]", URI: "gs://[BUCKET_NAME]/[FILE_NAME]"]
2024-05-31 16:01:48,488 DEBUG    urllib3.connectionpool Starting new HTTPS connection (1): sqladmin.googleapis.com:443
2024-05-31 16:01:48,963 DEBUG    urllib3.connectionpool https://sqladmin.googleapis.com:443 "POST /sql/v1beta4/projects/durable-sound-424715-q2/instances/%5BINSTANCE_NAME%5D/import?alt=json HTTP/1.1" 400 None
2024-05-31 16:01:48,966 DEBUG    root            (gcloud.sql.import.csv) HTTPError 400: Invalid request: Invalid full instance name (durable-sound-424715-q2:[INSTANCE_NAME])..
Traceback (most recent call last):
  File "/snap/google-cloud-cli/239/lib/googlecloudsdk/calliope/cli.py", line 998, in Execute
    resources = calliope_command.Run(cli=self, args=args)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/snap/google-cloud-cli/239/lib/googlecloudsdk/calliope/backend.py", line 815, in Run
    resources = command_instance.Run(args)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/snap/google-cloud-cli/239/lib/surface/sql/import/csv.py", line 75, in Run
    return import_util.RunCsvImportCommand(args, client)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/snap/google-cloud-cli/239/lib/googlecloudsdk/command_lib/sql/import_util.py", line 235, in RunCsvImportCommand
    return RunImportCommand(args, client, csv_import_context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/snap/google-cloud-cli/239/lib/googlecloudsdk/command_lib/sql/import_util.py", line 149, in RunImportCommand
    result_operation = sql_client.instances.Import(import_request)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/snap/google-cloud-cli/239/lib/googlecloudsdk/generated_clients/apis/sqladmin/v1beta4/sqladmin_v1beta4_client.py", line 832, in Import
    return self._RunMethod(
           ^^^^^^^^^^^^^^^^
  File "/snap/google-cloud-cli/239/lib/third_party/apitools/base/py/base_api.py", line 737, in _RunMethod
    return self.ProcessHttpResponse(method_config, http_response, request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/snap/google-cloud-cli/239/lib/third_party/apitools/base/py/base_api.py", line 743, in ProcessHttpResponse
    self.__ProcessHttpResponse(method_config, http_response, request))
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/snap/google-cloud-cli/239/lib/third_party/apitools/base/py/base_api.py", line 609, in __ProcessHttpResponse
    raise exceptions.HttpError.FromResponse(
apitools.base.py.exceptions.HttpBadRequestError: HttpError accessing <https://sqladmin.googleapis.com/sql/v1beta4/projects/durable-sound-424715-q2/instances/%5BINSTANCE_NAME%5D/import?alt=json>: response: <{'vary': 'Origin, X-Origin, Referer', 'content-type': 'application/json; charset=UTF-8', 'content-encoding': 'gzip', 'date': 'Fri, 31 May 2024 16:01:48 GMT', 'server': 'ESF', 'cache-control': 'private', 'x-xss-protection': '0', 'x-frame-options': 'SAMEORIGIN', 'x-content-type-options': 'nosniff', 'transfer-encoding': 'chunked', 'status': 400}>, content <{
  "error": {
    "code": 400,
    "message": "Invalid request: Invalid full instance name (durable-sound-424715-q2:[INSTANCE_NAME])..",
    "errors": [
      {
        "message": "Invalid request: Invalid full instance name (durable-sound-424715-q2:[INSTANCE_NAME])..",
        "domain": "global",
        "reason": "invalid"
      }
    ]
  }
}
>
2024-05-31 16:01:48,975 ERROR    root            (gcloud.sql.import.csv) HTTPError 400: Invalid request: Invalid full instance name (durable-sound-424715-q2:[INSTANCE_NAME])..
